core/src/com/unciv/ui/popups/options/ModCheckTab.kt

```diff
@@ -151,15 +152,17 @@ class ModCheckTab(
                         .thenBy { it.name }
                 )
             for (mod in modsToCheck) {
-                val baseForThisMod = if (base != MOD_CHECK_DYNAMIC_BASE) base else getBaseForMod(mod)
-                if (baseForThisMod == null) {
+                val dynamicBase = base == MOD_CHECK_DYNAMIC_BASE
```

Comment:
I think the term 'dynamic base' is the source of confusion here. We're talking about a modder-specified base, so why not say so? Something like DECLARED_REQUIRED_MODS or something.

Reply:
It sure confused me I'll tell you that, I would prefer this was changed
Not a blocker for this PR since it doesn't make it worse, but still... 'dynamic' to me means 'can change according to situations', but for each *mod* it's always the same...